frameworks/mastra/scrubbed-agent/AGENTS.md

```
@@ -0,0 +1,110 @@
+- The file should export a default function
+- Prefer naming the default function Agent or the name of the Agent based on the context of the Agent description
+- All code should be in TypeScript format
+- Use the provided logger from the `AgentContext` interface such as `ctx.logger.info("my message: %s", "hello")`
+
+### Example Agent File
+
+```typescript
+import type { AgentRequest, AgentResponse, AgentContext } from "@agentuity/sdk";
+
+export default async function Agent(req: AgentRequest, resp: AgentResponse, ctx: AgentContext) {
+	return resp.json({hello: 'world'});
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)